3444/0/triangle.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+
+# :reek:UtilityFunction and :reek:FeatureEnvy
+def first_ex(a_side, b_side, c_side)
```

Comment:
first ex? WTF?

Reply:
yea, it's looks horrible) thanks for review, need some sleep for going on